CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.10.1]
+### Fixed
+- limit click area in the checkbox component to be follow the width of the content
```

Comment:
please use `Checkbox`

to be follow -> to follow
the width of the content -> the content width